### ISSUE TITLE
Nethermind and Nimbus don't work together

### ISSUE THREAD

**Issue by thomkeh** [eth2-educators/eth2-docker#227]
I wanted to use some of the lesser-used clients to improve network diversity and picked Nethermind and Nimbus. But Nimbus can't seem to connect to Nethermind properly. It might be a bug in Nimbus, but I'm not sure.

I tried all the following settings:
```bash
ETH1_NODE=http://eth1:8545
# or
ETH1_NODE=http://eth1:8546
# or
ETH1_NODE=ws://eth1:8546
# or
ETH1_NODE=ws://eth1ws:8546
# or
ETH1_NODE=ws://eth1:8546/ws/json-rpc
```

But Nimbus still tells me it can't connect (this message appears over and over again):

```
INF 2021-05-04 21:14:49.114+00:00 Starting Eth1 deposit contract monitoring  topics="eth1" tid=1 file=eth1_monitor.nim:1006 contract=0x00000000219ab540356cbb839cbe05303d7705fa url=ws://eth1:8546
ERR 2021-05-04 21:14:49.116+00:00 Eth1 chain monitoring failure, restarting  topics="eth1" tid=1 file=eth1_monitor.nim:1104 err="Trying to access value with err: Failed to setup web3 connection"
```

(On the other hand, Teku seems to work fine with Nethermind.) I suspect the problem is websockets somehow.

**Comment by yorickdowne** [comment]
I'll need to dig a little. Thank you for this report!

**Comment by yorickdowne** [comment]
For geth it's `ws://eth1:8546`. Just confirmed this works:
```
beacon_1            | INF 2021-05-11 17:41:15.118+00:00 Starting Eth1 deposit contract monitoring  topics="eth1" tid=1 file=eth1_monitor.nim:1006 contract=0xff50ed3d0ec03ac01d4c79aad74928bff48a7b2b url=ws://eth1:8546
beacon_1            | INF 2021-05-11 17:41:15.119+00:00 Scheduling first slot action               topics="beacnde" tid=1 file=nimbus_beacon_node.nim:1049 startTime=7w3h41m15s119ms782us657ns nextSlot=353907 timeToNextSlot=8s880ms217us343ns
beacon_1            | INF 2021-05-11 17:41:15.123+00:00 Waiting for new Eth1 block headers         topics="eth1" tid=1 file=eth1_monitor.nim:509
```

**Comment by natpicone** [comment]
> For geth it's `ws://eth1:8546`. Just confirmed this works:
> 
> ```
> beacon_1            | INF 2021-05-11 17:41:15.118+00:00 Starting Eth1 deposit contract monitoring  topics="eth1" tid=1 file=eth1_monitor.nim:1006 contract=0xff50ed3d0ec03ac01d4c79aad74928bff48a7b2b url=ws://eth1:8546
> beacon_1            | INF 2021-05-11 17:41:15.119+00:00 Scheduling first slot action               topics="beacnde" tid=1 file=nimbus_beacon_node.nim:1049 startTime=7w3h41m15s119ms782us657ns nextSlot=353907 timeToNextSlot=8s880ms217us343ns
> beacon_1            | INF 2021-05-11 17:41:15.123+00:00 Waiting for new Eth1 block headers         topics="eth1" tid=1 file=eth1_monitor.nim:509
> ```

I'm using
`ETH1_NODE=ws://eth1:8546,wss://eth-goerli.ws.alchemyapi.io/v2/XXXXXXXXXX`

And I got:
```
beacon_1                    | INF 2021-05-11 19:08:02.293+00:00 Starting Eth1 deposit contract monitoring  topics="eth1" tid=1 file=eth1_monitor.nim:1006 contract=0xff50ed3d0ec03ac01d4c79aad74928bff48a7b2b url=ws://eth1:8546,wss://eth-goerli.ws.alchemyapi.io/v2/XXXXXXXXXX
beacon_1                    | ERR 2021-05-11 19:08:02.293+00:00 Eth1 chain monitoring failure, restarting  topics="eth1" tid=1 file=eth1_monitor.nim:1104 err="Trying to access value with err: Failed to setup web3 connection"
```

**Comment by yorickdowne** [comment]
This may be resolved with Nethermind 1.10.70. Can you try that again with `ETH1_NODE=ws://eth1:8546` and Nimbus and Nethermind, @thomkeh ?

**Comment by yorickdowne** [comment]
Closing this as I believe it to have been solved with a Nethermind release.
